### PR TITLE
[vi]: Update translations

### PR DESCRIPTION
**Pull request by datlechin**

### CODE REVIEW

File: locales/vi/json.json [hunk 1]
```diff
@@ -204,8 +204,8 @@
     "Done.": "Xong.",
     "Download": "Tải Về",
     "Download Receipt": "Tải Về Hóa Đơn",
-    "Drop file or click to choose": "Drop file or click to choose",
-    "Drop files or click to choose": "Drop files or click to choose",
+    "Drop file or click to choose": "Kéo thả hoặc nhấp để chọn tệp tin",
+    "Drop files or click to choose": "Kéo thả hoặc nhấp để chọn tệp tin",
```
Comment (dinhquochan) [inline locales/vi/json.json:208]: Use `click`, this word could not be translated.

File: locales/vi/json.json [hunk 2]
```diff
@@ -434,7 +434,7 @@
...
     "Once your account is deleted, all of its resources and data will be permanently deleted. Before deleting your account, please download any data or information that you wish to retain.": "Khi tài khoản của bạn bị xóa, tất cả tài nguyên và dữ liệu của tài khoản đó sẽ bị xóa vĩnh viễn. Trước khi xóa tài khoản của bạn, vui lòng tải xuống bất kì dữ liệu hoặc thông tin nào bạn muốn giữ lại.",
-    "Once your account is deleted, all of its resources and data will be permanently deleted. Please enter your password to confirm you would like to permanently delete your account.": "Once your account is deleted, all of its resources and data will be permanently deleted. Please enter your password to confirm you would like to permanently delete your account.",
+    "Once your account is deleted, all of its resources and data will be permanently deleted. Please enter your password to confirm you would like to permanently delete your account.": "Khi tài khoản của bạn bị xoá, tất cả tài nguyên và dữ liệu của tài khoản đó sẽ bị xoá vĩnh vieienx. Vui lòng nhập mật khẩu của bạn đã xác nhận nếu bạn muốn xoá vĩnh viễn tài khoản của bạn.",
```
Comment (dinhquochan) [inline locales/vi/json.json:437]:
```suggestion
    "Once your account is deleted, all of its resources and data will be permanently deleted. Please enter your password to confirm you would like to permanently delete your account.": "Khi tài khoản của bạn bị xoá, tất cả tài nguyên và dữ liệu của tài khoản đó sẽ bị xoá vĩnh viễn. Vui lòng nhập mật khẩu của bạn đã xác nhận nếu bạn muốn xoá vĩnh viễn tài khoản của bạn.",
```

File: locales/vi/json.json [hunk 3]
```diff
@@ -571,7 +571,7 @@
     "Sint Maarten (Dutch part)": "Sint Maarten (phần tiếng Hà Lan)",
     "Slovakia": "Slovakia",
     "Slovenia": "Slovenia",
-    "Soft Deleted": "Soft Deleted",
+    "Soft Deleted": "Xoá tạm thời",
```
Comment (dinhquochan) [inline locales/vi/json.json:574]:
```suggestion
    "Soft Deleted": "Xoá Tạm Thời",
```

File: locales/vi/json.json [hunk 4]
```diff
@@ -718,7 +718,7 @@
     "United States": "Hoa Kỳ",
     "United States Minor Outlying Islands": "Đảo nhỏ xa xôi hẻo lánh của Hoa Kỳ",
     "United States Outlying Islands": "Các quần đảo xa xôi của Hoa Kỳ'",
-    "Unknown": "Unknown",
+    "Unknown": "Không xác định",
```
Comment (dinhquochan) [inline locales/vi/json.json:721]:
```suggestion
    "Unknown": "Không Xác Định",
```

**Review by dinhquochan — CHANGES_REQUESTED** [review]

**Review by dinhquochan — APPROVED** [review]
LGTM.

**Review by andrey-helldar — APPROVED** [review]
Cảm ơn

**Pull request merged**